src/Simplify.elm

```diff
@@ -15619,30 +15619,27 @@ alwaysResultsInConstantError :
     -> Error {}
 alwaysResultsInConstantError usingSituation config checkInfo =
     let
-        addNecessaryParens : String -> String
-        addNecessaryParens string =
-            if config.replacementNeedsParens then
-                "(" ++ string ++ ")"
-
-            else
-                string
-
         replacement : QualifyResources {} -> String
-        replacement =
+        replacement res =
             case checkInfo.argCount - (1 + List.length checkInfo.argsAfterFirst) of
```

Comment:
Probably worth extracting this value we're pattern matching on, so we don't recompute it twice (since `replacement` is called twice).

Reply:
It's called with different resources, not sure there's much we can do